library/Backends/llvm/src/Juvix/Backends/LLVM/Pass/Types.hs

```diff
@@ -164,4 +164,17 @@ data TermLLVM
   | AppM
       (Annotated TermLLVM)
       [Annotated TermLLVM]
+  | ScopedRecordDeclM RecordDecl (Annotated TermLLVM)
+  | FieldM TermFieldSelector
+  | RecordM TermRecordConstructor
```

Comment:
We should comment if these are new.

Reply:
Done in https://github.com/anoma/juvix/pull/1106/commits/442dfcd63ac8e61e0e60e28d013edcf25e70fc8f